### PR TITLE
Doc page dialog component

### PR DESCRIPTION
**Pull request by Jialecl**

### CODE REVIEW

File: website/screens/components/dialog/usage/DialogUsagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          />
+        </Figure>
+        <DxcText as="p">
+          The overlay element makes possible to get the user attention into the dialog creating a
+          layer between the actual application and the modal information showed in the user
+          interface.
+        </DxcText>
+      </DxcStack>
+
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-style-guide/blob/master/website/screens/overview/introduction/IntroductionPage.tsx" />
```
Comment (marcialfps) [inline website/screens/components/dialog/usage/DialogUsagePage.tsx:70]: Github link is not correct.

File: website/screens/components/dialog/usage/DialogUsagePage.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import Image from "next/image";
+import { DxcList, DxcStack, DxcText } from "@dxc-technology/halstack-react";
+import HeadingLink from "../../../common/HeadingLink";
+import Code from "../../../common/Code";
+import dialogContent from "./images/dialog_content.png";
+import dialogOverlay from "./images/dialog_overlay.png";
+import DocFooter from "../../../common/DocFooter";
+import Figure from "../../../common/Figure";
+
+const DialogUsage = () => {
```
Comment (marcialfps) [inline website/screens/components/dialog/usage/DialogUsagePage.tsx:10]: Rename it to `DialogUsagePage`.

File: website/screens/components/dialog/usage/DialogUsagePage.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import Image from "next/image";
+import { DxcList, DxcStack, DxcText } from "@dxc-technology/halstack-react";
+import HeadingLink from "../../../common/HeadingLink";
+import Code from "../../../common/Code";
```
Comment (marcialfps) [inline website/screens/components/dialog/usage/DialogUsagePage.tsx:4]: Remove unused import.

File: website/screens/components/dialog/specs/DialogSpecsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,451 @@
+import Image from "next/image";
+import dialogAnatomyImage from "./images/dialog_anatomy.png";
+import dialogSpecsImage from "./images/dialog_specs.png";
+import { DxcList, DxcStack, DxcTable, DxcText } from "@dxc-technology/halstack-react";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+import Figure from "../../../common/Figure";
+import HeadingLink from "../../../common/HeadingLink";
+
+const DialogSpecs = () => {
```
Comment (marcialfps) [inline website/screens/components/dialog/specs/DialogSpecsPage.tsx:10]: Rename to `DialogSpecsPage`.

File: website/screens/components/dialog/specs/DialogSpecsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,451 @@
...
+              <td>
+                <Code>xxlarge</Code>
+              </td>
+              <td>100px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcText as="p">
+          And also apply different values to each side of the component:
+          <Code>top</Code> <Code>bottom</Code> <Code>left</Code> <Code>right</Code>
```
Comment (marcialfps) [inline website/screens/components/dialog/specs/DialogSpecsPage.tsx:443]: Add commas.

File: website/screens/components/dialog/specs/DialogSpecsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,451 @@
...
+        </DxcText>
+      </DxcStack>
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-style-guide/blob/master/website/screens/overview/introduction/IntroductionPage.tsx" />
```
Comment (marcialfps) [inline website/screens/components/dialog/specs/DialogSpecsPage.tsx:446]: Github link is not correct.

**Review by marcialfps — CHANGES_REQUESTED** [review]
Where is Accesibility section?? 😮

File: website/screens/components/dialog/specs/DialogSpecsPage.tsx [hunk 7]
```diff
@@ -0,0 +1,476 @@
...
+          <Code>bottom</Code>, <Code>left</Code> and <Code>right</Code>
+        </DxcText>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Accesibility</HeadingLink>
+        <HeadingLink level={4}>WCAG</HeadingLink>
+        <DxcList>
+          <DxcText>
+            Understanding WCAG 2.2 -{" "}
+            <DxcLink href="https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap.html" text="2.1.2 No Keyboard Trap" />
```
Comment (marcialfps) [inline website/screens/components/dialog/specs/DialogSpecsPage.tsx:452]: Add to external links `newWindow` prop, please.

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: website/screens/components/dialog/specs/DialogSpecsPage.tsx (hunk 7 above)
Comment (Jialecl) [reply website/screens/components/dialog/specs/DialogSpecsPage.tsx]: it is done!

**Review by marcialfps — APPROVED** [review]

**Pull request merged**